### PR TITLE
Add LDU information to seeds

### PR DESCRIPTION
**Pull request by KPobeeNorris**
We have implemented the responsibility override feature that allows
SPOs to change the responsibility from custody to community.  However,
so that the team can use this feature we need to add some LDU
information into the seeds file; this PR does that.

### CODE REVIEW

**Comment by rossjones** [comment]
Might want to add an LDU with no email address as well?

**Comment by KPobeeNorris** [comment]
New LDU without email address added

File: db/seeds.rb [hunk 1]
```diff
@@ -32,3 +32,64 @@
...
+# The offenders below are those with release dates a few years in the future and can therefore use the
+# responsibility override workflow
+
+CaseInformation.find_or_create_by!(
+    nomis_offender_id: 'G7658UL',
+    tier: 'A',
+    case_allocation: 'NPS',
+    welsh_offender: 'Yes',
+    manual_entry: true,
+    local_divisional_unit_id: 1,
```
Comment (rossjones) [inline db/seeds.rb]: Rather than assume id of 1, might it be best to use the result of find_or_create_by! above to use that id?

Comment (KPobeeNorris) [reply db/seeds.rb]: Implemented

**Pull request merged**